hathor/cli/top.py

```diff
@@ -24,774 +24,760 @@
 from math import floor
 from typing import Any, Callable, DefaultDict, Dict, List, Optional, Tuple
 
-try:
+# XXX: as annoying as it is, a simple `if: raise` is not enough, but putting the whole module inside works
+if sys.platform != 'win32':
```

Comment:
I prefer to raise an exception and let the importer method handle it instead of doing it. So, I think we should remove this if statement. What do you think?

Reply:
I prefer that too! But, although mypy correctly understands `sys.platform`, it doesn't understand that a global exception should make it ignore the rest of the file. So the only simple solution I found was to put everything inside an `if`. I also tried having both the `if` and a `raise`, like `if sys.platform == 'win32': raise Exception; else: ...`, but that also makes mypy complaint. Maybe there a better solution, I just don't want to spend too much time on this.

Reply:
Should we move the code to `_top.py` and here just do the following?

```python
if sys.platform != 'win32':
    from _top import *
```

Reply:
That also doesn't work. Because mypy will type-check `_top.py` and make the same complaints it makes here. Also that is exactly how the `top` module is used in `hathor/cli/main.py`:

```python
# ...
        if sys.platform != 'win32':
            from . import top
            self.add_cmd('hathor', 'top', top, 'CPU profiler viewer')
# ...
```

Reply:
To be honest, I hate not failing on Windows. But I agree that we can refactor it later. Just to confirm: Except for the indentation, no other change was made on this file, right?

Reply:
Yes, the only change is the indentation.